students/ghassan/lesson09/helpers.py

```diff
@@ -0,0 +1,12 @@
+def main_menu():
+    choice = input(
+        'Pick one:\n1) Send a thank you\n2) Create a Report\n3) Send letters to everyone\n4) Quit\n'
+        )
+    return choice
+
+
+def set_donor_name():
```

Comment:
it would make more sense to call it `get_donor_name` since you're getting a value instead of setting

Reply:
Yes, I agree.